addons/event_booth_sale/models/sale_order.py

```diff
@@ -23,6 +24,13 @@ def _compute_event_booth_count(self):
     def action_confirm(self):
         res = super(SaleOrder, self).action_confirm()
         for so in self:
+            so_lines_missing_booth = self.env['sale.order.line']
+            for line in so.order_line:
+                if line.product_type == 'event_booth' and not line.event_booth_pending_ids:
+                    so_lines_missing_booth |= line
+            if so_lines_missing_booth:
+                event_names = "".join(["- " + event.name + "\n" for event in so_lines_missing_booth])
```

Comment:
For these you can use fstrings instead of concatenation, so:

`[f"- {event.name} \n" for event in so_lines_missing_booth]`

Also is it really "event names" in this case? It's the name of the SO line, not sure we can name that "event", right? Should be more something like "so_lines_descriptions" ?